library/build.gradle

```diff
@@ -46,21 +46,25 @@ android {
             dimension "tweaksMode"
         }
     }
+    lint {
+        sarifReport true
+        checkDependencies true
+    }
 }
 
 dependencies {
 
-    enabledImplementation 'androidx.core:core-ktx:1.7.0'
-    enabledImplementation 'androidx.appcompat:appcompat:1.4.0'
-    enabledImplementation 'com.google.android.material:material:1.4.0'
-    enabledImplementation "androidx.compose.material:material:$compose_version"
+    enabledImplementation 'androidx.core:core-ktx:1.9.0'
+    enabledImplementation 'androidx.appcompat:appcompat:1.5.1'
+    enabledImplementation 'com.google.android.material:material:1.7.0'
+    enabledImplementation "androidx.compose.material:material:1.3.1"
```

Comment:
Why not extract version numbers like compose_version?

Reply:
Because `compose_version` is 1.3.2, but there is no such release in material (don't know why)

Reply:
I meant to extract version numbers to variables in base build.gradle file

Reply:
sorry, I've integrated the PR, maybe we can do that, yes. I was keeping it simply with the versions that aren't shared across different libraries because this project is small, but they can be extracted, of course :)